stetho/src/main/java/com/facebook/stetho/inspector/network/AsyncPrettyPrinterExecutorHolder.java

```diff
@@ -16,5 +16,21 @@
  * A holder class for the executor service used for pretty printing related tasks
  */
 class AsyncPrettyPrinterExecutorHolder {
-  public static final ExecutorService sExecutorService = Executors.newCachedThreadPool();
+  private static ExecutorService sExecutorService;
+
+  static void ensureInitialized() {
+    if (sExecutorService == null) {
+      sExecutorService = Executors.newCachedThreadPool();
+    }
+  }
+
+  public static ExecutorService getExecutorService() {
```

Comment:
the return value cannot be null here since I called "ensureInitialized" before returning an executor service.